File: vpmm_Server/manager.py
import pill


class PillManager:

    def __init__(self):
        self.pill_list = []

    def add_medicine(self, medicine_name, medicine_qty, before_meal, ):
        self.pill_list.append(medicine_name)

    def view_medicine(self):
        pass

    def update_medicine(self):
        pass

    def delete_medicine(self):
        pass


p = pill.Pill("Percocet", 10, True, 1, 12, False)

File: medicare/lib/manager.dart
class PillManager {
  void addMedicine() {}
  void viewMedicine() {}
  void updateMedicine() {}
  void deleteMedicine() {}
}

class Pill {
  String name;
  int totalQty;
  bool beforeOrAfterMeal;
  int intakeQty;
  int intakeTime;
  bool intakeDaily;
  List<bool> selectedDays;

  Pill(this.name, this.totalQty, this.beforeOrAfterMeal, this.intakeQty,
      this.intakeTime, this.intakeDaily, this.selectedDays);
}
